issue_test.go

```diff
@@ -29,3 +45,15 @@ func TestIssueToJSON(t *testing.T) {
 
 	assert.Equal(t, expectedJSON, string(issueAsJSON))
 }
+
+func TestIssueWithSuggestionToJSON(t *testing.T) {
+	issue := testIssueWithSuggestion()
+
+	issueAsJSON, err := issue.ToJSON()
+	if err != nil {
+		t.Error("Failed converting to JSON")
+	}
```

Comment:
Since this is a test, if the marshaling fails the test will fail, so we can simplify:
```suggestion
	issueAsJSON, _ := issue.ToJSON()
```